Added information for section 3.1 and 3.4 in the UG. (#337)
AY2122S1-CS2103T-W16-1/tp: docs/UserGuide.md

## 3. About

This section aims to help familiarise you with how our application works as well as the terminologies that we use

> throughout our application and user guide.

Reviewer: hmm not sure if we need our application

Reviewer 2: also i believe its terminology

Reviewer 2: I think its fine leh

### 3.1 Special Icons

> Here are the special icons used throughout the User Guide to help you utilise the User Guide more effectively.

Reviewer: Here are the special icons used throughout our User Guide to help you utilise this guide more effectively.

#### Tips

Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.

<div markdown="span" class="alert alert-primary">:bulb:
Tip: Tips are helpful!
</div><br>

#### Notes

> Notes are here to provide the User with extra information that they may find useful when using DonnaFin.

Reviewer: provide you with extra information

<div markdown="span" class="alert alert-primary">:information_source:
Take notes when you see this icon!

#### Caution 

> Cautions are around to warn users of potential unwanted side effects that may occur on certain parts of DonnaFin if the 

Reviewer: warn you of potential unwanted side effects that may occur when using certain features of DonnaFin if you fail to follow this guide properly or tamper with the files of the application without adequate knowledge.

user fails to follow the user guide or tamper with the files of the application.

<div markdown="span" class="alert alert-primary">
:exclamation: Be wary of cautions!
</div><br>

### 3.2 Application Home Window Breakdown

### 3.3 Application Client Window Breakdown

#### 3.3.4 Asset/Liabilities Tab

###### This section covers both the asset and liability tab due to their similarities.

The Asset/Liabilities tab is the Client View tab that stores all of your client's assets/liabilities. This tab can be
accessed by pressing on the Assets/Liabilities tab when in any of the Client View tabs, or through typing the command

> `tab a`, `tab asset`, or `tab assets` for the Asset tab and typing the command `tab l`, `tab liability`, or

Reviewer: Assets Tab
You also forgot to add Liabilities tab after the tab commands

`tab liabilites`.

> You will know that you are in the Asset tab if you see the **Assets** header below the Result Display or the Liability

Reviewer: Assets tab and Liabilities tab

tab if you see the **Liabilities** header below the Result Display. You can see the total value of these assets/
liabilities by looking at the number beside the **Total Asset/Liability Value**. 

The Assets/Liabilities are displayed in a Table View where each row represents a new asset/liability, while each column

> represents a Field of an Asset/Liability.

Reviewer: not sure if field has to be in caps

![Asset/Liabilities Tab of DonnaFin](images/Asset_Liability.png)

Here are the following fields that each asset/liability records:
1. Index
2. Asset/Liability Name
3. Asset/Liability Type
4. Asset/Liability Value
5. Remarks

Each asset/liability has an index which is sorted by ascending order, and if you would like to remove an existing
asset/liability , you can keep track of the index by looking at the number to the left of the asset/liability name you
would like to delete.

The Asset/Liability tab also supports the `append` and `remove` command that allows you to add a new asset/liability or
remove an existing asset/liability as shown 

> [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#4331-add-an-asset-to-a-client-append.)

Reviewer: tbh [here.](#4331-add-an-asset-to-a-client-append.) is sufficient

Author: wait the square brackets disappear in my preview ah

Reviewer: no like use #4331-add-an-asset-to-a-client-append inside the round brackets no need the http stuff

Reviewer 2: ```suggestion
[here](#4331-add-an-asset-to-a-client-append).
```

#### 3.3.5 Notes Tab

> The Notes tab is the Client View tab that stores additional information about your client. This tab can be accessed by

Reviewer: Client Window tab? consider : The Notes tab stores additional information about your client.

> pressing on the `Notes button` when in any of the Client View tabs, or through typing the command `tab n`, `tab notes`

Reviewer: `Notes` button when you are in any of the Client Window tabs

## 3. About

> This section aims to help familiarise you with how our application works as well as the terminologies that we use

Reviewer 2: ```suggestion
This section aims to help familiarize you with how our application works as well as the terminology that we use
```

throughout our application and user guide.

### 3.1 Special Icons

> Here are the special icons used throughout the User Guide to help you utilise this guide more effectively.

Reviewer: our User Guide

Reviewer 2: ```suggestion
Here are the special icons used throughout our User Guide to help you utilise this guide more effectively.
```

#### Tips

Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.

<div markdown="span" class="alert alert-primary">:bulb:
Tip: Tips are helpful!
</div><br>

#### Notes

> Notes are here to provide you with extra information that they may find useful when using DonnaFin.

Reviewer 2: ```suggestion
Notes are here to provide you with extra information that you may find useful when using DonnaFin.
```

<div markdown="span" class="alert alert-primary">:information_source:
Take notes when you see this icon!

#### Caution 

Cautions are around to warn you of potential unwanted side effects that may occur when using certain features of
DonnaFin if you fail to follow this guide properly or tamper with the files of the application without adequate
knowledge.

<div markdown="span" class="alert alert-primary">

> :exclamation: Be wary of cautions!

Reviewer 2: ```suggestion
:exclamation: Here lies a cautionary tale!
```

hehe a joke (yes shee hui, its lame)
